charactersheet/charactersheet/services/common/account/user_service.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+/**
+ * A global service that fetches the user's account information.
+ */
+var UserServiceManager = new SharedServiceManager(_UserService, null);
+
+/**
+ * An internal service implementation that holds onto data regarding the
+ * account login status, tokens and expiration.
+ *
+ * This service contains all of the relevant information for keeping
+ * track of a user's authentication state.
+ */
+function _UserService(config) {
+    var self = this;
+
+    self.url = '/api/users.json';
```

Comment:
I think there are around 4 separate API endpoints in this PR. How would fetching the API endpoints dynamically work?